src/main/java/org/jenkinsci/plugins/ewm/steps/ExwsExecution.java

```diff
@@ -97,6 +106,25 @@ public boolean start() throws Exception {
         return false;
     }
 
+    /**
+     * TODO JAVADOC
+     * @param workspaceId
+     * @throws IOException
+     */
+    private void registerFingerprint(String workspaceId) throws IOException {
+        FingerprintMap map = Jenkins.getActiveInstance().getFingerprintMap();
+        Fingerprint f = map.get(workspaceId);
+        if (f == null) {
+            return;
```

Comment:
Should it throw exception? Seems in such case the method does not do what it was expected